pkg/controller/controller.go

```diff
@@ -279,7 +279,7 @@ func (c *controller) getClusterServiceClassPlanAndClusterServiceBroker(instance
 
 	broker, err := c.brokerLister.Get(serviceClass.Spec.ClusterServiceBrokerName)
 	if err != nil {
-		s := fmt.Sprintf("ServiceInstance \"%s/%s\" references a non-existent broker %q", instance.Namespace, instance.Name, serviceClass.Spec.ClusterServiceBrokerName)
+		s := fmt.Sprintf("ServiceInstance \"%s/%s\": references a non-existent broker %q", instance.Namespace, instance.Name, serviceClass.Spec.ClusterServiceBrokerName)
```

Comment:
Needs backticks and `%s`->`%v` for consistency.

Reply:
These are wrong. This is for controller.go, I suppose the <type> is CatalogController? Maybe I revert the changes to this file.